usecases/replica/repairer.go

```diff
@@ -54,19 +56,54 @@ func (r *repairer) repairOne(ctx context.Context,
 	contentIdx int,
 ) (_ *storobj.Object, err error) {
 	var (
+		deleted   bool
 		lastUTime int64
 		winnerIdx int
 		cl        = r.client
 	)
 	for i, x := range votes {
 		if x.o.Deleted {
-			return nil, errConflictExistOrDeleted
+			deleted = true
+			break
 		}
 		if x.UTime > lastUTime {
 			lastUTime = x.UTime
 			winnerIdx = i
 		}
 	}
+
+	if deleted {
+		if r.objectDeletionConflictResolution != models.ReplicationConfigObjectDeletionConflictResolutionPermanentDeletion {
+			return nil, errConflictExistOrDeleted
+		}
+
+		gr := enterrors.NewErrorGroupWrapper(r.logger)
```

Comment:
shouldn't we set a limit here to the number of goroutines invoked?

Reply:
aham, just followed the same pattern was done when the objects are updated rather than deleted.  the number of goroutines would be at max `replication factor`-1  of the class. One per replica that needs to receive a change for the object in question